test/colony-network-mining.js

```diff
@@ -675,24 +703,25 @@ contract("ColonyNetworkMining", accounts => {
 
     it("should prevent submission of hashes with an invalid entry for the balance of a user", async () => {
       await giveUserCLNYTokensAndStake(colonyNetwork, MAIN_ACCOUNT, DEFAULT_STAKE);
-      await giveUserCLNYTokensAndStake(colonyNetwork, OTHER_ACCOUNT, DEFAULT_STAKE);
-      const repCycle = await getActiveRepCycle(colonyNetwork);
 
+      const repCycle = await getActiveRepCycle(colonyNetwork);
       await forwardTime(MINING_CYCLE_DURATION, this);
+
       await checkErrorRevert(
         repCycle.submitRootHash("0x12345678", 10, 1000000000000, { from: MAIN_ACCOUNT }),
         "colony-reputation-mining-stake-minimum-not-met-for-index"
       );
-      const entryNumber = await getValidEntryNumber(colonyNetwork, MAIN_ACCOUNT, "0x87654321");
-      await repCycle.submitRootHash("0x87654321", 10, entryNumber, { from: OTHER_ACCOUNT });
+
+      await repCycle.submitRootHash("0x87654321", 10, 10, { from: MAIN_ACCOUNT });
     });
 
     it("should prevent submission of hashes with a valid entry, but invalid hash for the current time", async () => {
       await giveUserCLNYTokensAndStake(colonyNetwork, MAIN_ACCOUNT, DEFAULT_STAKE);
 
       const repCycle = await getActiveRepCycle(colonyNetwork);
+
       await checkErrorRevert(
-        repCycle.submitRootHash("0x12345678", 10, 1, { from: MAIN_ACCOUNT }),
+        repCycle.submitRootHash("0x12345678", 10, 10, { from: MAIN_ACCOUNT }),
```

Comment:
We normally have `entryIndex` as 10 in the tests, and IMO semantically, breaks with convention communicate intention. If the test itself is neutral to the value of `entryIndex`, we should stick with the convention to communicate that "this isn't part of the test".

Reply:
Since this test is demonstrating that a submission at the beginning of the mining cycle will (almost surely) fail regardless of the nonce, I think it's best to communicate that the `entryIndex` doesn't matter by leaving it to the conventional value of `10`.